CRM/Upgrade/Incremental/sql/5.15.alpha1.mysql.tpl

```diff
@@ -9,3 +9,22 @@ SELECT @option_group_id_ps as option_group_id, {localize field='label'}`label`{/
 FROM civicrm_option_value ov
 INNER JOIN civicrm_option_group og
 ON og.id = ov.option_group_id AND og.name = 'contribution_status';
+
+SELECT @maxValue := MAX(value) FROM `civicrm_option_value` where option_group_id = @option_group_id_ps;
+SELECT @maxWeight := MAX(weight) FROM ROM `civicrm_option_value` where option_group_id = @option_group_id_ps;
+
+INSERT INTO `civicrm_option_value` (
+`option_group_id`, {localize field='label'}`label`{/localize}, `value`, `name`, `weight`, `is_reserved`, `is_active`, `is_default`
```

Comment:
Inserting new entries into optiongroup tables is the single most common cause for failed upgrades (for sites I've upgraded).  Can we check if they already exist before inserting?

Reply:
@mattwire I added the ON DUPLICATE to the end - normally I'd use 'ensureOptionValueExists' at the php layer but since we've only just created the group I'm not sure I can